Sources/Fuzzilli/Core/ProgramBuilder.swift

```diff
@@ -829,10 +829,19 @@ public class ProgramBuilder {
                 self.splice(from: program)
             }, {
                 // We can't run code generators if we don't have any visible variables.
-                guard self.scopeAnalyzer.visibleVariables.count > 0 else { return }
-                let generator = self.fuzzer.codeGenerators.randomElement()
-                if generator.requiredContext.isSubset(of: self.context) {
-                    self.run(generator)
+                if self.scopeAnalyzer.visibleVariables.count == 0 {
+                    // Generate some variables
+                    self.run(self.fuzzer.programPrefixGenerators.randomElement()!)
```

Comment:
Nit: I think you can use `chooseUniform(from: self.fuzzer.programPrefixGenerators)` to avoid the `!`